web/src/buddy_list.js

```diff
@@ -97,45 +422,148 @@ export class BuddyList extends BuddyListConf {
         this.update_padding();
     }
 
+    should_hide_headers(current_sub, pm_ids_set) {
+        // If we have only "other users" and aren't in a stream/DM view
+        // then we don't show section headers and only show one untitled
+        // section.
+        return this.users_matching_view_ids.length === 0 && !current_sub && !pm_ids_set.size;
+    }
+
+    render_view_user_list_links() {
+        const {current_sub, subscriber_count, other_users_count} = this.render_data;
+        const has_inactive_users_matching_view =
+            subscriber_count > this.users_matching_view_ids.length;
+        const has_inactive_other_users = other_users_count > this.other_user_ids.length;
+
+        // For stream views, we show a link at the bottom of the list of subscribed users that
+        // lets a user find the full list of subscribed users and information about them.
+        if (
+            current_sub &&
+            stream_data.can_view_subscribers(current_sub) &&
+            has_inactive_users_matching_view
+        ) {
+            const stream_edit_hash = hash_util.stream_edit_url(current_sub, "subscribers");
+            $("#buddy-list-users-matching-view-container").append(
+                render_view_all_subscribers({
+                    stream_edit_hash,
+                }),
+            );
+        }
+
+        // We give a link to view the list of all users to help reduce confusion about
+        // there being hidden (inactive) "other" users.
+        if (has_inactive_other_users) {
+            $("#buddy-list-other-users-container").append(render_view_all_users());
+        }
+    }
+
     get_items() {
-        const $obj = this.$container.find(`${this.item_selector}`);
-        return $obj.map((_i, elem) => $(elem));
+        const $user_matching_view_obj = this.$users_matching_view_container.find(
+            `${this.item_selector}`,
+        );
+        const $users_matching_view_elems = $user_matching_view_obj.map((_i, elem) => $(elem));
+
+        const $other_user_obj = this.$other_users_container.find(`${this.item_selector}`);
+        const $other_user_elems = $other_user_obj.map((_i, elem) => $(elem));
+
+        return [...$users_matching_view_elems, ...$other_user_elems];
     }
```

Comment:
I guess this isn't new, but do you understand the purpose of the `.map` here? I would have thought that we already had jQuery objects in the `.find` result.

Hmm is this function dead code? I don't see any relevant matches to `git grep get_items`.

Reply:
Great catch, it looks like it became dead in the `compose fade users: Remove fade logic in favor of buddy list split` commit. I'll another commit directly after it to remove `get_items`.